docs/configuration/index.md

```diff
@@ -168,7 +168,7 @@ If `celeborn.worker.flusher.buffer.size` is 256 KB, we can have total slots up t
 
 ## Rack Awareness
 
-Celeborn can be rack-aware by setting `celeborn.client.reserveSlots.rackware.enabled` to `true` on client side.
+Celeborn can be rack-aware by setting `celeborn.client.reserveSlots.rackaware.enabled` to `true` on client side.
```

Comment:
```
  val CLIENT_RESERVE_SLOTS_RACKAWARE_ENABLED: ConfigEntry[Boolean] =
    buildConf("celeborn.client.reserveSlots.rackaware.enabled")
      .withAlternative("celeborn.client.reserveSlots.rackware.enabled")
      .categories("client")
      .version("0.3.1")
      .doc("Whether need to place different replicates on different racks when allocating slots.")
      .booleanConf
      .createWithDefault(false)
```

Reply:
Since 0.3.1, should we use the new config?

Reply:
We can either keep this or remove it. However, if we decide to remove it, we will need to update the migration guide accordingly.

Reply:
ok, will keep it